MapboxCoreNavigation/Feedback.swift

```diff
@@ -0,0 +1,5 @@
+import Foundation
+
+public enum FeedbackType: String {
+    case general = "general"
```

Comment:
It’s currently impossible for an Objective-C application to call `RouteController.recordFeedback(type:description:)` et al., because this public type doesn’t bridge to Objective-C.